tests/app/invite/test_invite_rest.py

```diff
@@ -115,6 +115,7 @@ def test_get_all_invited_users_by_service(client, notify_db, notify_db_session,
     for invite in json_resp['data']:
         assert invite['service'] == str(sample_service.id)
         assert invite['from_user'] == str(invite_from.id)
+        assert invite['auth_type'] == SMS_AUTH_TYPE
```

Comment:
Should the test method now be test_get_all_invited_users_via_sms_by_service

Reply:
nah i haven't changed the method, it's just that users default to `SMS_AUTH` - i'm checking to make sure it's returned to the front-end as that info is needed by the admin